### ISSUE TITLE
transfer denied due to blocklimit, but no blocklimit set

### ISSUE THREAD

**Issue by kontu** [LordTylus/SE-Torch-Plugin-ALE-PcuTransferrer#4]
!transfer "ACME CEO" "Omicron to Delta"

```
Player 'ACME CEO' does not have a high enough Blocklimit!
Max: 0
Built: 8837
After Transfer: 11716
Vertified with: Vanilla Limits
```

From the SE-Dedi config :
```
<MaxBlocksPerPlayer>0</MaxBlocksPerPlayer>
    <TotalPCU>100000</TotalPCU>
    <PiratePCU>50000</PiratePCU>
    <MaxFactionsCount>0</MaxFactionsCount>
    <BlockLimitsEnabled>NONE</BlockLimitsEnabled>
```

**Comment by LordTylus** [comment]
Nice find,

I believe I never tested it with blocklimits completely disabled. Or it used to work and I recently broke it... who knows.

Well a workaround for that is to use the !fircetransfer command instead. However I think I may be able to fix it this weekend.

**Comment by LordTylus** [comment]
Fixed in 1.1.8.1
